testing_script/variables.py

```diff
@@ -1,22 +1,23 @@
 acceptCookie = "hs-eu-confirmation-button"
 careersButton = "Careers"
 locationDrobBox = "//span[@class='label' and text()='Location']"
-selectingVilnius = "//*[@id='post-108']/div/div[1]/div[2]/div[1]/div/div[1]/div[3]/div/ul/li[10]"
-categoryDropBox = "//*[@id='post-108']/div/div[1]/div[2]/div[1]/div/div[2]/div[2]/b"
-selectingSales = '//*[@id="post-108"]/div/div[1]/div[2]/div[1]/div/div[2]/div[3]/div/ul/li[6]'
-searchForJobs = "//button[contains(text(), 'Search')]"
-technologyAndEngineeringCheckBox = "//*[@id='search-filter']/div[4]/div/div[4]"
-seniorTestAutomationEngineerPosition = 'Senior'
-frameName = 'grnhse_iframe'
-inputName = 'first_name'
-inputLastName = 'last_name'
-inputMail = 'email'
-inputPhone = 'phone'
-uploadCv = '//*[@id="resume_fieldset"]/div/div[3]/button[1]'
-uploadCoverLetter = '//*[@id="cover_letter_fieldset"]/div/div[3]/button[1]'
-coverLetterUploadedImg = '//*[@id="cover_letter_chosen"]/button/img'
-agreeWithPolicy = '//*[@id="data_compliance"]/div/label'
-submitFormButton = 'submit_buttons'
+selectingVilnius = "//li[contains(text(), 'Vilnius')]"
```

Comment:
https://www.w3schools.com/python/python_variables_names.asp
Bet gerai ir kaip tu rasai. Nenustebk kad ateityje rays kitaip, nes del patogumo buna kad nutaria is anksto kokia stilistika bus naudojama, nes tai priklauso nuo projekto